### PR TITLE
Added Homepage screen

### PR DESCRIPTION
**Pull request by shivanigangadharan**
Deploy Preview for the Homepage screen : https://deploy-preview-3--starlit-praline-d4418c.netlify.app/homepage

### CODE REVIEW

File: src/components/sidebar/sidebar.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import '../../styles.css';
+import './sidebar.css';
+import defaultImg from '../../assets/defaultImg.png';
+
+export default function Sidebar() {
+    return (
+        <div className="sidebar-container">
+            <div className="menu-options">
+                <div><i className="fa-solid fa-house"></i> Home</div>
```
Comment (sharathnair9999) [inline src/components/sidebar/sidebar.js]: for icons u need not use `div`. `span` would be enough. keep it semantic 😃

File: src/pages/homepage/homepage.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import '../../styles.css';
+import './homepage.css';
+import Sidebar from '../../components/sidebar/sidebar';
```
Comment (sharathnair9999) [inline src/pages/homepage/homepage.js:4]: try to put all the components in an `index.js` file in the components folder and import the components from there. would be better for readability

**Review by sharathnair9999 — APPROVED** [review]
Suggested minute changes. Good work

**Review by dev-enforced — APPROVED** [review]
The live preview looks good as mentioned in the PR.
Had a minor query related to the position of the account details on the sidebar
account_position_query
Like when we scroll the webpage this section should remain fixed at the bottom right so that the user can click on it to go to the profile route? Or is there some other working behind it?
Apart from this query rest of the code looks structured
Do apply the suggestions given by Sharath related to importing the files from a single folder through index.js format. That will help you import multiple pages in a single line itself!!
You can use code formatters such as Prettier too to make the code readable for reviewers!!
Good work @shivanigangadharan 😀

File: src/pages/homepage/homepage.js (hunk 2 above)
Comment (shivanigangadharan) [reply src/pages/homepage/homepage.js:4]: Done, thanks for mentioning this.

File: src/components/sidebar/sidebar.js (hunk 1 above)
Comment (shivanigangadharan) [reply src/components/sidebar/sidebar.js]: Done 👍🏼

**Comment by shivanigangadharan** [comment]
> The live preview looks good as mentioned in the PR. Had a minor query related to the position of the account details on the sidebar account_position_query Like when we scroll the webpage this section should remain fixed at the bottom right so that the user can click on it to go to the profile route? Or is there some other working behind it? Apart from this query rest of the code looks structured Do apply the suggestions given by Sharath related to importing the files from a single folder through index.js format. That will help you import multiple pages in a single line itself!! You can use code formatters such as Prettier too to make the code readable for reviewers!! Good work @shivanigangadharan 😀

Done, fixed the sidebar. Thanks for suggesting this one. 👍🏼

**Pull request merged**